roles/openshift_cfme/tasks/main.yml

```diff
@@ -0,0 +1,176 @@
+# PASSED IN ROLE VARIABLES:
+#
+# - template_dir - A remote temp dir to template the PVs in
+
+---
+- name: What are the groups?
```

Comment:
Debugging so I can see what all the correct `oo_*` group names are when the `std_include.yml` file isn't commented out.